package.json

```diff
@@ -30,7 +30,7 @@
     "link-dev": "mkdir -p .bin/ && cd packages/amplify-cli && ln -s \"$(pwd)/bin/amplify\" ../../.bin/amplify-dev && cd ../../",
     "link-win": "node ./scripts/link-bin.js packages/amplify-cli/bin/amplify amplify-dev",
     "lint-check-package-json": "yarn eslint --no-eslintrc --config .eslint.package.json.js '**/package.json'",
-    "lint-check": "yarn eslint . --ext .js,.jsx,.ts,.tsx --max-warnings=835",
+    "lint-check": "yarn eslint . --ext .js,.jsx,.ts,.tsx --max-warnings=773",
```

Comment:
The warning count is reduced because `amplify-e2e-core` is now subject to `test` linting rules.
Most of this was `@typescript-eslint/no-explicit-any`. Which seems to be reasonable sacrifice to simplify test configuration.